### ISSUE TITLE
FREEZING, LAGGING

### ISSUE THREAD

**Issue by woodssowild** [al-caughey/Google-Meet-Attendance#53]
I am a teacher at an elementary school. I am using a laptop from my school for online classes using Google Classroom, meeting with Google meet. I added Google Meet Attendance to my Google Classrooms.  My laptop was working fine, but has begun lagging/freezing. I believe this started after installing Attendance. I tried turning off Attendance, but lagging has continued. Is it possible Attendance is causing this?

**Comment by Eagle-Man** [comment]
When you say "turning off Attendance," do you mean you turned off monitoring (the toggle switch in the extension) or that you disabled the extension itself.  If it's the latter, then there's no way that it's causing the lagging.  If the extension is disabled, then it can't be affecting the browser (as far as I know).

If it's the former, then...I guess it's possible, but no one else has mentioned a problem.

**Comment by woodssowild** [comment]
Yes, I turned off monitoring (toggle switch). To disable it, must I remove it? Can I temporarily disable it without removing it so I can see what happens? As you can probably tell, this kind of thing is not my strong point.

**Comment by Eagle-Man** [comment]
Nope.  Go to chrome://extensions in the address bar (or Menu->More Tools->Extensions) and you can use the toggle there to disable it without uninstalling.

**Comment by woodssowild** [comment]
I'll give it a try, thanks.

**Comment by AlCaughey** [comment]
What other extensions do you have installed?
Do you have Grid View or Grid View (fix)?  If the former, replace it with
the latter.

No doubt, my extension will add some load to the Meet (when it is active)
but much of that should disappear if you hit the toggle button in the top
right corner of the attendance dialog before starting the Meet.  If you are
still getting a lag, it is (IMHBO), it is likely being caused by something
else.  But I'm more than willing to accept that that might not be entirely
true

Al
